lib/virt_autotest/utils.pm

```diff
@@ -567,4 +568,40 @@ EOF
     return;
 }
 
+=head2 check_port_state
+
+Check whether given port is open on remote machine. This subroutine accepts four
```

Comment:
kudos for the nice explanatory perldoc. just one thinky. Add `check_port_state($dst_machine, $dst_port, $retries, $delay)` after `head2`

Reply:
Done

Reply:
I think the suggestion might have been ambiguous. Please see https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L154 for an example how this should be structured

```
=head2 check_port_state

    check_port_state($dst_machine, $dst_port, $retries, $delay)

Check whether given port is open on remote machine. This subroutine accepts four
```

Reply:
sorry for confusion

Reply:
Done